README.md

```diff
@@ -1,307 +1,51 @@
 
-```sh
-make build-installer IMG=<some-registry>/oadp-nac:tag
-```
+## Getting Started
 
-NOTE: The makefile target mentioned above generates an 'install.yaml'
-file in the dist directory. This file contains all the resources built
-with Kustomize, which are necessary to install this project without
-its dependencies.
+### Prerequisites
+- go version v1.21.0+
+- oc
+- Access to a OpenShift cluster
+- [OADP operator](https://github.com/openshift/oadp-operator) installed in the cluster
 
-2. Using the installer
+> **TODO:** Today, OADP operator must be installed in `openshift-adp` namespace, because of this limitation https://github.com/migtools/oadp-non-admin/blob/master/internal/controller/nonadminbackup_controller.go#L51
 
-Users can just run kubectl apply -f <URL for YAML BUNDLE> to install the project, i.e.:
+### Using NAC
 
-```sh
-kubectl apply -f https://raw.githubusercontent.com/<org>/oadp-nac/<tag or branch>/dist/install.yaml
-```
+To use NAC functionality:
+- as admin user, create non admin user (to create a non admin user to test NAC, check [non admin user documentation](docs/non_admin_user.md)) and its namespace
+- as admin user, create/update DPA and set non admin feature to enabled
+- as non admin user, create non admin CRs
 
 ## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
-
-**NOTE:** Run `make help` for more information on all potential `make` targets
 
-More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
+Please check our [contributing documentation](docs/CONTRIBUTING.md) to propose changes to the repository.
 
 ## Architecture
 
-The project was generated using kubebuilder version `v3.14.0`, running the following commands
-```sh
-kubebuilder init \
-    --plugins go.kubebuilder.io/v4 \
-    --project-version 3 \
-    --project-name=oadp-nac \
-    --repo=github.com/migtools/oadp-non-admin \
-    --domain=oadp.openshift.io
-kubebuilder create api \
-    --plugins go.kubebuilder.io/v4 \
-    --group nac \
-    --version v1alpha1 \
-    --kind NonAdminBackup \
-    --resource --controller
-make manifests
-```
-> **NOTE:** The information about plugin and project version, as well as project name, repo and domain, is stored in [PROJECT](PROJECT) file
-
-To upgrade kubebuilder version, create kubebuilder structure using the current kubebuilder version and the upgrade version, using the same commands presented earlier, in two different folders. Then generate a `diff` file from the two folders and apply changes to project code.
-
-## Quality
-
-The quality checks of the project are reproduced by the continuos integration (CI) pipeline of the project. CI configuration in [`.github/workflows/ci.yml`](.github/workflows/ci.yml) file.
+For a better understanding of the project, check our [architecture documentation](docs/architecture.md).
 
-To run all checks locally, run `make ci`.
+## OADP version compatibility
```

Comment:
IMHO we do not need this, right ? We would not be doing separate NAC releases that are decoupled from OADP Operator release.

Reply:
we will probably not do releases

but this compatibility information I do believe is important to know where the code comes from and to test 

For example, when 1.4 release branch is created, this needs to be changed there https://github.com/migtools/oadp-non-admin/pull/16/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R260

Reply:
removed